src/Test/L0/DotnetsdkDownloadScriptL0.cs

```diff
@@ -12,7 +12,7 @@ public sealed class DotnetsdkDownloadScriptL0
         [Trait("Category", "Agent")]
         public async Task EnsureDotnetsdkBashDownloadScriptUpToDate()
         {
-            string shDownloadUrl = "https://raw.githubusercontent.com/dotnet/cli/master/scripts/obtain/dotnet-install.sh";
```

Comment:
i am not interesting on change this, the original string can tells you the actual script is on a github repo, and when you are hitting a problem you can open an issue on their repository. :)

Reply:
I'm going to bring that one separately.

Reply:
Brought it to #2031 with a better description.